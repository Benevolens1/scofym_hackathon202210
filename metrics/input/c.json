{"1":{
	"type": "typeA",
	"arguments": {"arg1": "Obra",
		"arg2": "Guga"}
	},
"2": {
	"type": "typeB",
	"arguments": {"arg1": "Prima"}
},
"3":{
	"type": "typeA",
	"arguments": {"arg1": "Dirot",
		"arg2": "pufff",
        "arg3": "key"}
	},
"4":{
    "type": "typeB",
    "arguments": {}
}
}
