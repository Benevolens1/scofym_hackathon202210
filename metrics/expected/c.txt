4 Not filled !
1 Obra_Guga
2 Prima
3 Dirot_pufffkey
